libraries/core-react/src/Banner/BannerActions.tsx

```diff
@@ -1,12 +1,16 @@
-// @ts-nocheck
-import React from 'react'
-import PropTypes from 'prop-types'
+import React, { HTMLAttributes, ReactNode } from 'react'
 import styled from 'styled-components'
 import { banner as tokens } from './Banner.tokens'
 
 const { enabled } = tokens
 
-const StyledBannerActions = styled.div`
+type BannerActionsPlacement = 'bottom' | 'left'
```

Comment:
Not sure if this is a good way to reduce code duplication? Was considering making a types file, but I am not sure what it should look like.

Reply:
For now we have defined them in the component they are used. If there are any that are shared across sub-components. We have normally just created a `types.ts`. We have a rollup plugin that extract types for dist, but are anyways planning to have revision on types when we finish rewriting as to better see what can be shared and not.